app/models/manageiq/providers/lenovo/physical_infra_manager/refresher.rb

```diff
@@ -1,9 +0,0 @@
-module ManageIQ::Providers::Lenovo
-  class PhysicalInfraManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
```

Comment:
@d-m-u I think we should keep the `Lenovo::PhysicalInfraManager::Refresher` class even if it is empty

Reply:
cool, just a second